RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -294,6 +294,72 @@ boolean LX_getStarKey()
 
 boolean beehiveConsider()
 {
+	int damage_sources = 1; // basic hit
+	
+	// Familiars
+	if (have_familiar($familiar[shorter-order cook]) && auto_is_valid($familiar[shorter-order cook]))
+	{
+		damage_sources += 5;
+	}
+	else if (have_familiar($familiar[mu]) && auto_is_valid($familiar[mu]))
+	{
+		damage_sources += 5;
+	}
+	else if (have_familiar($familiar[imitation crab]) && auto_is_valid($familiar[imitation crab]))
+	{
+		damage_sources += 4;
+	}
+	
+	// Prismatic damage sources
+	if (possessEquipment($item[june cleaver])&&auto_is_valid($item[june cleaver]))
+	{
+		damage_sources += 5;
+		if (have_skill($skill[double-fisted skull smashing]))
+		{
+			damage_sources += 1;
+		}
+	}
+	else if (possessEquipment($item[giant bow tie])&&auto_is_valid($item[giant bow tie]))
+	{
+		damage_sources += 5;
+		if (have_skill($skill[double-fisted skull smashing]))
+		{
+			damage_sources += 1;
+		}
+	}
+	else if (possessEquipment($item[roman candelabra])&&auto_is_valid($item[roman candelabra]))
+	{
+		damage_sources += 4; // doesn't do spooky, so let's get that from a spookyraven skill
+		if (have_skill($skill[Snarl of the Timberwolf]) || have_skill($skill[Dirge of Dreadfulness]))
+		{
+			damage_sources += 1;
+		}
+	}
+	
+	// Combat skill to use
+	if (have_skill($skill[kneebutt]))
```

Comment:
`auto_combat_default_stage1.ash` does not use kneebutt, but headbutt instead. Probably should use kneebutt, I presume the headbutt is to avoid fumbles (it also uses clobber, which is worse than attacking with an offhand).